app/Http/Controllers/Api/LicenseSeatsController.php

```diff
@@ -136,7 +138,12 @@ public function update(Request $request, $licenseId, $seatId) : JsonResponse | a
         if ($licenseSeat->save()) {
 
             if ($is_checkin) {
-                $licenseSeat->logCheckin($target, $request->input('note'));
+                if(!$licenseSeat->license->reassignable){
+                    $licenseSeat->notes .= "\n" .trans('admin/licenses/message.checkin.not_reassignable') . ".";
+                    $licenseSeat->unreassignable_seat = true;
+                    $licenseSeat->save();
+                }
```

Comment:
Can you add a test for this behavior? 

`Feature/Checkins/Api/LicenseCheckinTest::testUnreassignableLicenseSeatMarkedUponCheckin()` or something.